src/software/ai/hl/stp/tactic/pass_defender/pass_defender_fsm.h

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#include "shared/constants.h"
+#include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/logger/logger.h"
+
+struct PassDefenderFSM
+{
+    class BlockPassState;
+    class InterceptBallState;
+
+    // This struct defines the unique control parameters that the PassDefenderFSM requires
+    // in its update
+    struct ControlParams
+    {
+        // The location on the field to block enemy passes from
+        Point position_to_block_from;
+    };
+
+    DEFINE_TACTIC_UPDATE_STRUCT_WITH_CONTROL_AND_COMMON_PARAMS
+
+    // The minimum speed of the ball for it to be considered a pass
+    static constexpr double MIN_PASS_SPEED = 0.5;
```

Comment:
I wonder if this is too small and results in false positives where the enemy robot is dribbling the ball at a speed of >0.5m/s and we think there's been a pass. Curious what you think? Not sure if it can be easily simulated, but might be an interesting scenario for a simulated test

Reply:
I remember that something similar came up a while ago in #2542 regarding `ball_not_kicked_threshold` for `ShootOrPassPlayFSM` and I don't think we had a conclusive answer.